### DISCUSSION TITLE
Run dependent job after its dependency fails

### DISCUSSION THREAD

**Discussion by k-sap** [rq/rq#1622] [Q&A]
Hi,
I'd like to as if there is possibility to run a job after the job it is dependent on fails.

I have a use case when I prefer operations to be done sequentially. However, after an error I'd like the second dependent task to run.

So basically, to change the criteria of depends_on from completion to just being finished in any way.

**Comment by ccrvlh** [comment]
@k-sap have you tried looking into the `allow_failure` dependency attribute? Would help with this issue?
